src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,22 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_mutator_plugin(extra_env):
+  """Decide whether to use Radamsa in process. If yes, add the path to the
+  radamsa shared object to LD_PRELOAD in |extra_env| and return True."""
+
+  # Mutator Plugin doesn't work on windows
+  if environment.platform() == "WINDOWS":
```

Comment:
As-is this should probably bail out on any non-LINUX values. We'll also want to validate that this is an asan job for the time being (based on Jonathan's previous comment).

Reply:
And i386

Reply:
I changed it so that it bails out of any job that isn't Linux or isn't ASAN. Is that correct?

Reply:
I 3 8 6